apps/lottery/src/components/index.ts

```diff
@@ -0,0 +1,4 @@
+import { Header, Footer, Content } from './layout';
```

Comment:
I suggest to export without import

Reply:
[Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript)